sdk/translation/Azure.AI.Translation.Text/src/Azure.AI.Translation.Text.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the TextTranslator client library for developing .NET applications with rich experience.</Description>
+    <AssemblyTitle>Azure SDK Code Generation TextTranslator for Azure Data Plane</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>Azure TextTranslator</PackageTags>
```

Comment:
Consider making this

<PackageTags>Azure TextTranslator;$(PackageCommonTags)</PackageTags>

Reply:
Done